extra-libs/openexr/autobuild/prepare

```diff
@@ -0,0 +1,5 @@
+CMAKE_AFTER+="""
```

Comment:
Why declare it here?

Reply:
`ABPY3VER` and `ABPY2VER` are unavailable in `defines`.